src/config/test.ts

```diff
@@ -1,8 +1,12 @@
 export default {
+  AWSAccessKeyId: 'AKIAJHFYB27I74EO26NQ',
+  AWSSecretKey: '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',
```

Comment:
**NEVER** put the AWS keys into the CSV especially into the poublic repo.

Reply:
Unfortunately, I cannot remove the keys right now as we need them for tests.

Reply:
Please use ENV vars set in the CI for this. The keys **must never** go public.